docs/SOPs/contributor_matrix_addition_SOP.md

```diff
@@ -27,8 +27,25 @@ Where either the expression matrix or cell type annotations cannot be found, the
 
 ## Filling in metadata about the files
 
+For each expression matrix or cell type annotation file that is found, a row needs to be filled in the metadata spreadsheet, in the ‘Analysis file’ tab. Analysis files can be linked to sequence file or biomaterial entities via the ‘Analysis_protocol’ entity (see the ‘Analysis protocol’ tab). This functions similar to a ‘sequencing protocol’ to link different entities. 
+
+The best practice is to link the analysis files to sequence file entities, if possible. Alternatively, you can also link the analysis files to cell suspension entities. This is currently done by adding the ‘Input Cell Suspension ID’ column to the ‘Analysis File’ tab and adding the linked cell suspensions to the cell.
+
+The gene expression matrix and cell annotations files should be added to the S3 bucket in the ingest-area together with raw data files, for instructions on how to use hca-util to do this, see ['here'](https://github.com/ebi-ait/hca-documentation/wiki/How-to-upload-data-to-an-upload-area-using-hca-util)
+
+![image](https://user-images.githubusercontent.com/68549377/117455950-96a30a80-af3f-11eb-895c-413d2fa89e2f.png)
```

Comment:
I think it's better to upload the image to the repo and link to it, I think if your user was for some reason deleted this image would be deleted as well